.github/workflows/gh-pages.yml

```diff
@@ -3,7 +3,7 @@ name: github pages
 on:
   push:
     branches:
-      - main  # Set a branch to deploy
+      - redesign  # Set a branch to deploy
```

Comment:
if this branch `xanimo:redesign` is being merged into the `main` branch of this repo, this should be changed back to `main` right?

Reply:
yes, but i'd like it to be it's own branch while Misha, Richard et al work on the dogepedia since this is a single page app now as oppose to the current one having page redirects.

Reply:
got it, that makes sense. if i understand correctly, this PR should be merged into `dogecoinfoundation:redesign` instead of `dogecoinfoundation:main`?

Reply:
yeah that's what i just proposed to the website channel. we'll see what's agreed upon.